src/utils/thirdparty_env.js

```diff
@@ -0,0 +1,26 @@
+import getResourcesPath from './resources'
+
+var thirdpartyPath = getResourcesPath('thirdparty'),
+  env = {};
+
+if (process.platform == 'darwin') {
+  // Set 3rd party binaries and libraries
+  env.PATH = [
+    "$PATH",
+    thirdpartyPath + "/poppler/bin",
+    thirdpartyPath + "/tesseract/bin"
+  ].join(':');
+
+  env.DYLD_LIBRARY_PATH = [
+    "$DYLD_LIBRARY_PATH",
+    "/System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/", // Core Graphics package
```

Comment:
`"/System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/"` should be `'/System/Library/Frameworks/ImageIO.framework/Versions/A/Resources/'`